src/pages/workspaces/workspace/Data.js

```diff
@@ -422,6 +422,26 @@ const DataTableActions = ({ workspace, tableName, rowCount, entityMetadata, onRe
   ])
 }
 
+const DataTableFeaturePreviewFeedbackBanner = () => {
+  const isDataTableProvenanceEnabled = isFeaturePreviewEnabled('data-table-provenance')
+  const isDataTableVersioningEnabled = isFeaturePreviewEnabled('data-table-versioning')
+
+  const label = _.join(' and ', _.compact([
+    isDataTableVersioningEnabled && 'versioning',
+    isDataTableProvenanceEnabled && 'provenance'
+  ]))
+  const feedbackUrl = `mailto:dsp-sue@broadinstitute.org?subject=${encodeURIComponent(`Feedback on data table ${label}`)}`
+
+  return (isDataTableProvenanceEnabled || isDataTableVersioningEnabled) && div({
+    style: {
+      padding: '1rem',
+      borderBottom: `1px solid ${colors.accent()}`,
+      background: colors.accent(0.2),
```

Comment:
This does not have sufficient color contrast.

![image](https://user-images.githubusercontent.com/484484/191967344-17290f09-f374-455c-b591-8af9e6c8dc89.png)

Reply:
I also found another issue on this page that I created a separate ticket for-- https://broadworkbench.atlassian.net/browse/SUP-747.